pkg/driver/controller.go

```diff
@@ -648,7 +657,7 @@ func (cs *controller) CreateSnapshot(
 
 func getSnapSize(params *SnapshotParams, capacity int64) int64 {
 	var snapSize int64
-	if !params.AbsSnapSize {
+	if !params.AbsSnapSize && params.SnapSize != 0 {
```

Comment:
do we need this change?

Reply:
No, since in both the case with nil params size will become 0 , so not required